scriptmodules/emulators/scummvm.sh

```diff
@@ -17,7 +17,12 @@ rp_module_section="opt"
 rp_module_flags=""
 
 function depends_scummvm() {
-    local depends=(libmpeg2-4-dev libogg-dev libvorbis-dev libflac-dev libmad0-dev libpng12-dev libtheora-dev libfaad-dev libfluidsynth-dev libfreetype6-dev zlib1g-dev libjpeg-dev)
+    local depends=(libmpeg2-4-dev libogg-dev libvorbis-dev libflac-dev libmad0-dev libtheora-dev libfaad-dev libfluidsynth-dev libfreetype6-dev zlib1g-dev libjpeg-dev)
+    if isPlatform "vero4k"; then
+        depends+=(vero3-userland-dev-osmc libpng-dev)
+    else
+        depends+=(libpng12-dev)
+    fi
```

Comment:
would prefer to handle the png logic as mentioned before - using the automatic system (set the raspbian version to 9 and it should work automatically). I will be changing all libpng12-dev to libpng-dev in the future (and put a workaround in for older systems to use libpng12-dev)